project/Frontend.scala

```diff
@@ -103,11 +103,9 @@ object Frontend extends Build with Prototypes {
 
   val admin = application("admin").dependsOn(commonWithTests).aggregate(common).settings(
     libraryDependencies ++= Seq(
-      postgres,
       paClient,
       dfpAxis,
       anormModule,
-      jdbc,
```

Comment:
I might have been delete happy here, is `jdbc` used? It compiles and runs locally fine.